packages/pigeon/run_tests.sh

```diff
@@ -130,6 +130,20 @@ gen_ios_unittests_code() {
     --objc_source_out platform_tests/ios_unit_tests/ios/Runner/$name.gen.m
 }
 
+snake_to_pascal_case() {
```

Comment:
New tests should be added to run_tests.dart, not run_tests.sh, as the intent is to migrate to run_tests.dart.

Reply:
Removed